chaos_genius/core/utils/data_loader.py

```diff
@@ -23,6 +23,8 @@
 
 
 class DataLoader:
+    """Data Loader Class."""
+
     def __init__(
         self,
         kpi_info: dict,
```

Comment:
The types below this need to have `Optional[]` since they can be None

Reply:
Done.